libnetwork/netavark/run.go

```diff
@@ -54,7 +55,7 @@ func (n *netavarkNetwork) Setup(namespacePath string, options types.SetupOptions
 	}
 
 	result := map[string]types.StatusBlock{}
-	err = n.execNetavark([]string{"setup", namespacePath}, netavarkOpts, &result)
+	err = n.execNetavark([]string{"--config", n.networkRunDir, "--rootless", strconv.FormatBool(n.networkRootless), "setup", namespacePath}, netavarkOpts, &result)
```

Comment:
--rootless should just be a bool option, e.g. there should be no extra argument for this.

Reply:
@Luap99 This is just to prevent unnecessary branching `if/else` in the code `--rootless` works as well.

Reply:
Does clap parser support the same syntax as cobra, e.g. for a bool option in podman you can still use `--rootless=true|false` to avoid branching?

Reply:
Yes `--rootless=` also works made a change to use `=` instead of `space`.